custom_components/tahoma/climate_aeh.py

```diff
@@ -87,3 +90,15 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
     async def async_turn_off(self) -> None:
         """Turn off the device."""
         await self.async_execute_command(COMMAND_SET_HEATING_LEVEL, PRESET_OFF)
+
+    async def async_down(self, **_):
```

Comment:
Is this required in all climate integrations?

Reply:
I don't know what happens if you call a service on an entity that doesn't have the method.